### DISCUSSION TITLE
Missing exe files after coapp installation

### DISCUSSION THREAD

**Discussion by silentpup** [aclap-dev/video-downloadhelper#8] [Help and Q&A]
### Your question

I have installed "Video DownloadHelper Companion App 2.0.10" ,
When I look at the "Video DownloadHelper" settings screen, it says "Companion App Not Installed" and I cannot use it.
Do you know how to solve this problem?

**Comment by jcfrog** [comment]
Hi,
Did you read this?  : https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized
jerome

**Reply by silentpup** [reply]
Yes, I read it.

On Windows, open a CMD shell, go to the installation directory and type 
. /vdhcoapp.exe --info and type.
Then, it said 
”'vdhcoapp.exe' is not recognized as an internal or external command,
operable program or batch file.”

The files in this directory are the following 4 files
google.json
microsoft.json
mozilla.json
uninstall.exe

Did I do something wrong in the procedure?

**Reply by paulrouget** [reply]
Do no use `./`. Just type "vdhcoapp.exe --info".

What browser do you use?

**Reply by silentpup** [reply]
Of course I did that.
My browser is firefox.

Attached is a capture of the work I have performed.
![not work](https://github.com/aclap-dev/video-downloadhelper/assets/25399727/a7836310-2175-4d80-98cd-2dfc8aad3e03)

**Comment by paulrouget** [comment]
Your directory is missing 3 files (vdhcoapp.exe, ffmpeg.exe, ffprobe.exe).

**Close Firefox**. And re-do the installation.

Let me know if that works.

Thank you for your patience.

**Comment by silentpup** [comment]
Thanks for your reply. 😊

Closed all browsers, and installed. But same result as before. 
I also deleted that directory once and ran it again, but the result was the same.😫

**Comment by paulrouget** [comment]
Do you see any errors while installing? (I imagine not, you would have mentioned it).

What about your antivirus, does it say anything?

If you install in another location, are the files still missing?

**Comment by silentpup** [comment]
I finally understood.😊

The intall location was wrong.😥

The correct directory is "net.downloadhelper.coapp" though,
The installer chose "Program Files".

Situation Description :
A few days ago, 
"Companion App" suddenly became unusable, so I installed "Companion App" according to the extension application,
I got this situation!

Anyway, the app worked.
It's all over
thanks! 😊

**Comment by paulrouget** [comment]
> net.downloadhelper.coapp

Wait… it should not be that. It's very strange.

When you run the `vdhcoapp.exe --info` command, what does it say?

**Comment by silentpup** [comment]
Okay.
I will show you, CMD window.

Is this something wrong?

![no title](https://github.com/aclap-dev/video-downloadhelper/assets/25399727/9a18c85e-dcf7-4833-8b73-52511f92ad9b)

**Reply by paulrouget** [reply]
ok - thank you! I understand now 👍 

All good.
